LimerickArtMap/LimerickArtMap/Services/ApiServices.cs

```diff
@@ -10,7 +10,7 @@ namespace LimerickArtMap.Services
 {
 	public class ApiServices
 	{
-		public async Task RegisterAsync(string email, string username, DateTime dateOfBirth, string password, string reconformPassword)
+		public async Task<bool> RegisterAsync(string email, string username, DateTime dateOfBirth, string password, string reconformPassword)
 		{
 			var client = new HttpClient();
 			var model = new UserAccount
```

Comment:
This is an empty duplicate UserAccount.  Get rid of it and import `LimerickStreetArt.UserAccount`

I suspect you will have to change LimerickStreetArt project target framework